Hippo.Web/ClientApp/src/Shared/Table/HipTableActionButton.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import HipButton from "../HipButton";
+import { ButtonProps } from "reactstrap";
+
+interface HipTableActionButtonProps extends ButtonProps {}
+
+const HipTableActionButton: React.FC<HipTableActionButtonProps> = ({
+  ...deferred
+}) => {
+  return (
+    <div className="data-table-prolog float-end">
+      <HipButton color="link" {...deferred}>
+        Create New Cluster
```

Comment:
seems pretty specific for a generically named action button?  did you want to allow passing in of the button content?